vac/raw/eth-secpm.md

```diff
@@ -824,6 +824,33 @@ These credentials MUST use the digital signature key pair associated to the Ethe
 - A Delivery Service storage system MUST verify KeyPackages before storing them.
 - Interaction diagrams involving the DS do not change.
 
+## Consideration related to the onchain component of the protocol
+- Each group is managed by a separate smart contract.
+
+- Below follows a description of Alice adding Bob as a member of a new group.
+    1. Alice creates the smart contract associated to the group. This smart contract MUST include an ACL.
+    2. Alice adds Bob’s Ethereum address to the ACL. 
+    3. Off-chain: Alice and Bob set a secure communication channel. This MUST be done following section *Private 1-to-1 communications protocol*
+    4. Off-chain: Bob sends a request to Alice so he can be included in the group. This request is a digitally signed message. **The signature MUST be generated using Bob’s secret key, associated to his Ethereum address.
+    5. Off-chain: Upon reception of the message, Alice verifies the signature and checks that it corresponds to an address contained in the ACL.
+    6. Off-chain: Alice sends a welcome message to Bob.
+    7. Off-chain: Once Bob gets the welcome message from Alice, he MUST send his keypackage and credentials to Alice.
+    8. Upon reception of Bob’s data, Alice registers it with the smart contract.
+    9. Off-chain: Alice SHOULD broadcasts a message anouncing the addition of Bob to other users of the group.
+
```

Comment:
Can you add a diagram for this flow, similar to what @seemenkina shared in discord?